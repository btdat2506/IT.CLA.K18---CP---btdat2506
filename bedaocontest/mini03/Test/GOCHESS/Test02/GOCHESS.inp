10 20
5 10 3
9 7 4
4 5 3
6 1 4
8 3 3
9 4 4
8 7 4
8 5 3
3 5 2
2 9 1
8 1 1
4 6 4
3 4 3
5 9 3
4 2 2
1 5 1
6 9 3
7 2 2
9 9 1
2 6 1
