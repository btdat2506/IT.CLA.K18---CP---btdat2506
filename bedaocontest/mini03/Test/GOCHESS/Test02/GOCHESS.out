1 2
4 3
5 4
6 3
3 5
7 4
6 4
6 4
2 4
0 2
0 5
4 2
2 4
4 2
2 3
1 3
4 3
5 2
1 4
2 2
